src/command/cmd_ac.c

```diff
@@ -1663,6 +1757,15 @@ cmd_ac_uninit(void)
     autocomplete_free(executable_ac);
     autocomplete_free(intype_ac);
     autocomplete_free(adhoc_cmd_ac);
+    autocomplete_free(vcard_ac);
+    autocomplete_free(vcard_photo_ac);
+    autocomplete_free(vcard_element_ac);
+    autocomplete_free(vcard_set_ac);
+    autocomplete_free(vcard_name_ac);
+    autocomplete_free(vcard_set_param_ac);
+    autocomplete_free(vcard_togglable_param_ac);
+    autocomplete_free(vcard_toggle_ac);
+    autocomplete_free(vcard_address_type_ac);
```

Comment:
autocomplete_reset() parts are missing

Reply:
oh i forgot

Reply:
https://codeberg.org/techmetx11/profanity/commit/619cb89c09683a4075bd62ae11667b1dd80b015a

Reply:
My friend! you created a GH account! i'm happy :) How about I close this PR and you create a new one (and we link to it) and then we review there? Then you can always push your changes on the PR.

Reply:
sure